### PR TITLE
I did it, I regret nothing

### LINKED ISSUE #2: Ruby client
As far as I've seen there is only a php and a python client but no ruby. Can we have a ruby client please?

### PR DESCRIPTION
**Pull request by Stelian**
Closes #2

### CODE REVIEW

File: stelian-ruby/raffler.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+if false == File.exist?(filePath) || false == File.readable?(filePath)
+    puts "The argument you passed is not a file or is not readable"
+    exit
+end
+
+# Ross also said we need it random, so don't return my name
+winner = File.readlines(filePath).sample
+
+# Return the random dude, make him happy
+puts "And the winner is " + winner
```
Comment (simicic) [inline stelian-ruby/raffler.rb]: I would add a bit of drama:  5.times { sleep(1); prints '.' }

Comment (Stelian) [inline stelian-ruby/raffler.rb]: I am a simple man, I can not handle drama but you should do that, and randomise drama!

**Comment by rosstuck** [comment]
+1 for drama. And tests. Cause Ruby.

**Comment by Stelian** [comment]
I do not agree with rules changing on the fly, now we do tests and drama?! I feel like my contributions to this project are not deemed worthy so I am going to go full tantrum mode here.

**Comment by simicic** [comment]
oh @rosstuck you could make a raffle competition, submit your raffle in the language of choice and the best one wins! ;) the price can be nothing but it is for the sake of fun :smile:

**Comment by rosstuck** [comment]
@simicic So meta. Yes.

The tests look good. We can merge this one but I really want to see one with randomized drama.

**Pull request merged**